packages/@sanity/core/src/actions/dataset/listDatasetCopyJobs.js

```diff
@@ -0,0 +1,90 @@
+module.exports = async function listDatasetCopyJobs(args, context) {
+  const {apiClient, output, chalk} = context
+  const flags = args.extOptions
+  const client = apiClient()
+  const projectId = client.config().projectId
+  const query = {}
+  let response
+
+  if (flags.offset && flags.offset >= 0) {
+    query.offset = flags.offset
+  }
+  if (flags.limit && flags.limit > 0) {
+    query.limit = flags.limit
+  }
+
+  try {
+    response = await client.request({
+      method: 'GET',
+      uri: `/projects/${projectId}/datasets/copy`,
+      query,
+    })
+  } catch (error) {
+    if (error.statusCode) {
+      output.print(`${chalk.red(`Dataset copy list failed:\n${error.response.body.message}`)}\n`)
```

Comment:
I also just double checked with the team and it's preferable to use `output.error()` over the `output.print` :)